cmake/modules/extensions.cmake

```diff
@@ -2524,59 +2558,41 @@ function(zephyr_get variable)
        ("${GET_VAR_SYSBUILD}" STREQUAL "GLOBAL" OR sysbuild_main_app)
     )
       get_property(sysbuild_${variable} TARGET sysbuild_cache PROPERTY ${variable})
```

Comment:
Shouldn't this (and the previous `${variable}` on line 2556) refer to `GET_VAR_VAR` instead now?

Reply:
it should.

Moved inside the `foreach()` and variable --> var